website/docs/style/icon/icon-a11y.md

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This guide should be rewritten, because there is no such role as icon, and our Icon component doesn't have all this roles and attributes you've added to the guide.